plugins/board-resources/src/customTypings/click.outside.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace svelte.JSX {
```

Comment:
Why we need this one?

Reply:
When I added custom `on:click_outside` handler to `div`, eslint showed error that `onclick_outside` wasn't defined on `HTMLDivElement`, removed now.